File: VB/TabHeaderDoubleClickEventArgs.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.Text
Imports System.Windows.Forms
Imports DevExpress.XtraTab

Namespace TabbedMDI
	Public Class TabHeaderDoubleClickEventArgs
		Inherits EventArgs
		Public Sub New(ByVal tabPage As IXtraTabPage)
			_TabPage = tabPage
		End Sub

		' Fields...
		Private _TabPage As IXtraTabPage

		Public ReadOnly Property TabPage() As IXtraTabPage
			Get
				Return _TabPage
			End Get
		End Property
	End Class
End Namespace

File: CS/TabHeaderDoubleClickEventArgs.cs
using System;
using System.Collections.Generic;
using System.Text;
using System.Windows.Forms;
using DevExpress.XtraTab;

namespace TabbedMDI
{
    public class TabHeaderDoubleClickEventArgs : EventArgs
    {
        public TabHeaderDoubleClickEventArgs(IXtraTabPage tabPage)
        {
            _TabPage = tabPage;
        }

        // Fields...
        private IXtraTabPage _TabPage;

        public IXtraTabPage TabPage
        {
            get { return _TabPage; }
        }
    }
}